package.json

```diff
@@ -9,7 +9,7 @@
     "@sap-cloud-sdk/http-client": "^3.2.0",
     "@sap-cloud-sdk/util": "^3.2.0",
     "@sap/audit-logging": "^5.0.0",
-    "@sap/cds": "^7.0.3",
+    "@sap/cds": "^8",
```

Comment:
`@sap/cds-hana` also needs to be replaced with `@cap-js/hana`. Please check with this new hana driver as well.